### PR TITLE
Implement colored avatars, ported Message.tsx & ConversationListItem.tsx to pure js

### PR DESCRIPTION
**Pull request by Jikstra**

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Merged https://github.com/deltachat/deltachat-desktop/pull/407 , so scratch the changes in `package.json`.

**Comment by Jikstra** [comment]
@r10s which one do you prefer?
screenshot_20181210_201737
screenshot_20181210_201802

**Comment by Jikstra** [comment]
> this is a bug on master, perhaps could be done in a separate PR?

Sure, but not really high priority in my opinion. Was the easiest to just do it here. Otherwise i probably will get some rebasing issues.

File: src/main/deltachat.js [hunk 1]
```diff
@@ -457,7 +457,10 @@ class DeltaChatController {
     const filemime = msg && msg.getFilemime()
     const fromId = msg && msg.getFromId()
     const isMe = fromId === C.DC_CONTACT_ID_SELF
-    const contact = fromId ? this._dc.getContact(fromId).toJson() : {}
+    let contact = fromId ? this._dc.getContact(fromId).toJson() : {}
+    if (contact.color) {
+      contact.color = '#' + contact.color.toString(16)
```
[the remark this answers is not in the document]
Comment (Jikstra) [reply src/main/deltachat.js]: Good idea

**Comment by karissa** [comment]
can we revert this? 

not sure I agree with the approach of putting it all into Message.js, it would be nice to keep a feature parity js version right next to the old one, perhaps just first do a port from ts to js first but still keep the file separate. Now in this PR we're requiring from ../../../(infinity)/conversations/build which seems a bit hacky, rather than keeping it a bit closer to each other.

**Comment by karissa** [comment]
its fine we don't have to revert, i'll work around it for now, but i feel like the structure does need a bit more thinking

**Pull request merged**